testcases/kernel/controllers/memcg/control/memcg_control_test.sh

```diff
@@ -116,7 +116,7 @@ result()
 cleanup()
 {
 	if [ -e $TST_PATH/mnt ]; then
-		umount $TST_PATH/mnt 2> /dev/null
+		umount -l $TST_PATH/mnt 2> /dev/null
 		rm -rf $TST_PATH/mnt
 	fi
 }
```

Comment:
If waiting for the pid does not fix the issue you should use ```tst_umount``` from the ```test.sh``` library.